server/chat-plugins/mafia.ts

```diff
@@ -1986,11 +1986,11 @@ export const pages: Chat.PageTable = {
 				buf += `<p><details><summary class="button" style="text-align:left; display:inline-block">Role Details</summary>`;
 				buf += `<table><tr><td style="text-align:center;"><img width="75" height="75" src="//${Config.routes.client}/fx/mafia-${role.image || 'villager'}.png"></td><td style="text-align:left;width:100%"><ul>${role.memo.map(m => `<li>${m}</li>`).join('')}</ul></td></tr></table>`;
 				buf += `</details></p>`;
-				if (game.dayNum > 1) {
-					for (let i = 1; i < game.dayNum; i++) {
-						previousActionsPL += `<b>Night ${i}</b><br/>`;
-						previousActionsPL += `${isPlayer.actionArr?.[i] ? `${isPlayer.actionArr[i]}` : ''}<br/>`;
-					}
+				for (let i = 0; i < game.dayNum; i++) {
+					previousActionsPL += `<b>Night ${i}</b><br/>`;
+					previousActionsPL += `${isPlayer.actionArr?.[i] ? `${isPlayer.actionArr[i]}` : ''}<br/>`;
+				}
+				if (game.phase === "day" || game.dayNum > 0) {
```

Comment:
Is the list of previous actions only supposed to appear during the day phase? Based on the original code, it appeared during both phases.

Reply:
Ok my bad, i missread the OR statement as AND, regardless this can be simplified, theres no reason to check if its day, just thats its not Night 0 because Day 0 isn't a thing.
```suggestion
				if (game.dayNum > 0) {
```